consensus/enclave/api/src/config.rs

```diff
@@ -0,0 +1,146 @@
+use crate::FeeMap;
+use alloc::{format, string::String};
+use mc_common::ResponderId;
+use mc_crypto_digestible::{Digestible, MerlinTranscript};
+use mc_transaction_core::BlockVersion;
+use serde::{Deserialize, Serialize};
+
+/// Configuration for the enclave which is used to help determine which
+/// transactions are valid.
+///
+/// (This can be contrasted with things like responder id and sealed block
+/// signing key)
+#[derive(Clone, Deserialize, Debug, Digestible, Eq, Hash, PartialEq, Serialize)]
+pub struct BlockchainConfig {
```

Comment:
It might be that a better name for this is `TxValidityConfig` or something like this